vault/vault.go

```diff
@@ -30,12 +30,19 @@ const (
 
 	AuthMethodKubernetes = "kubernetes"
 
-	AuthMethod              = "VAULT_AUTH_METHOD"
-	AuthMountPath           = "VAULT_AUTH_MOUNT_PATH"
-	AuthKubernetesRole      = "VAULT_AUTH_KUBERNETES_ROLE"
+	// AuthMethod is a vault authentication method used.
+	// https://www.vaultproject.io/docs/auth#auth-methods
+	AuthMethod = "VAULT_AUTH_METHOD"
+	// AuthMountPath defines a custom auth mount path.
+	AuthMountPath = "VAULT_AUTH_MOUNT_PATH"
```

Comment:
We need to explicitly call this out in docs to skip "auth/" when specifying this field